xml/templates/message_templates/membership_online_receipt_html.tpl

```diff
@@ -24,9 +24,8 @@
     {if !empty($receipt_text)}
      <p>{$receipt_text|htmlize}</p>
     {/if}
-
-    {if $is_pay_later}
-     <p>{$pay_later_receipt}</p> {* FIXME: this might be text rather than HTML *}
+    {if '{contribution.balance_amount:boolean}' === 'Pending' && {contribution.is_pay_later|boolean}}
```

Comment:
Is that first `contribution.balance_amount:boolean` correct? I thought it was contribution status name.

Reply:
hmm - it's not quite right - I used some copy & paste - I've swtiched to `{if {contribution.balance_amount|boolean}}`  - which should catch partially paid too

Reply:
Assuming that `contribution.balance_amount|boolean` works (which I assume it should), then that seems like it would be okay. Have not played with partial payment situation. Perhaps that would be a good thing to add to tests.

Reply:
Yeah - we have tests elsewhere on it as a token